.github/workflows/cla.yml

```
@@ -0,0 +1,65 @@
+name: "CLA Assistant"
+on:
+  pull_request_target:
+    types: [opened,closed,synchronize]
+
+# explicitly configure permissions, in case your GITHUB_TOKEN workflow permissions are set to read-only in repository settings
+permissions:
+  actions: write
+  contents: write
+  pull-requests: write
+  statuses: write
+
+jobs:
+  CLAAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "CLA Assistant"
+        if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
+        uses: contributor-assistant/github-action@v2.3.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # The below token has repo scope for the project configured further below to store signatures.
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.JUBIOTECH_CLA_SIGNATURES }}
+          PATH_TO_CLA: 'https://github.com/jubiotech/detl/blob/main/CLA.md'
+        with:
+          path-to-signatures: 'signatures_detl.json'
+          path-to-document: $PATH_TO_CLA
+          # branch should not be protected
+          branch: 'main'
+          allowlist: dependabot[bot]
+
+          # the followings are the optional inputs - If the optional inputs are not given, then default values will be taken
+          # 👇 the remote organization and repo where the signatures should be stored
+          remote-organization-name: jubiotech
+          remote-repository-name: cla-signatures
+          #create-file-commit-message: 'For example: Creating file for storing CLA Signatures'
+          #signed-commit-message: 'For example: $contributorName has signed the CLA in $owner/$repo#$pullRequestNo'
+          custom-notsigned-prcomment: >
+              Thank you for your submission, we really appreciate it!
+
+              Like many open-source projects we ask that you sign our [Contributor License Agreement](${{ env.PATH_TO_CLA }}) before we can accept your contribution.
+              To sign, please post two separate comments based on the following templates 👇
+
+              1. Comment:
+
+              ----
+
+              \- [ ] The JuBiotech maintainers know my real name.
+
+              At least one of the following two applies:
+
+              \- [ ] The JuBiotech maintainers know my current employer.
+
+              \- [ ] I am making not making this contribution on behalf of my current employer.

```

Shouldnt this be 

```
I am not making this contribution on behalf of my current employer.
```
?

---

No, if the contribution is made in the contributors free time, their current employer is irrelevant. They might also be self- or unemployed. Therefore only one of the two task items must apply.

---

I get that, I just think there is one *making* too many in the second point.

---

```suggestion
              \- [ ] I am making not this contribution on behalf of my current employer.

```
ah of course, sorry my in-mind-diffing was incomplete

---

I really think we need to consult Alexander Storm here. In my view, it makes a difference if somehone has an employer of if he/she made the contribution during working hours. To make things more complex, contributions in "free time" might still require permission of thge employer (or not). This is not our business but it must be clear to the contributor which option to choose. Some contributors (like Sam) have multiple accounts to clarify on/off work contributions.

---

Okay, then I will summarize the proposed procedure, texts & mockup screenshots, and consult him by email (cc y'all).